index.md

```diff
@@ -2,16 +2,83 @@
 layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
+date: '`r format(Sys.time(), "%d %B, %Y")`'
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you how to use ESMValTool.  
 
-{% comment %} This is a comment in Liquid {% endcomment %}
+The Earth System Model Evaluation Tool (ESMValTool) is a community developped software toolkit that aims to faciliate
+the diagnosis and evaluation of the causes and effects of model biases and inter-model spread within the CMIP model
+ensemble.
+
+
+This tutorial is structured such that the main body of the tutorial, up to the episode 7, can be done in one sitting.
+From there, there are several mini-tutorials which each covers one aspect of working with ESMValTool.
+These mini-tutorials can be appended to the main tutorial or worked through independently.
+
+> ## What will you learn in this course
+>
+>   - What is ESMValTool
+>
+>   - How to install ESMValTool
+>   - How to configure ESMValTool for your local system
+>   - How to run ESMValTool
+>   - How to work with ESMValTool's suite of preprocessors
+>   - How to debug your recipes
+>   - How to access and deploy recipes from the ESMValTools gallery (Advanced)
+>   - How to develop your own diagnostics and recipes (Advanced)
+>   - How to contribute your recipes and diagnostics back into ESMValTool (Advanced)
+>   - How to include new observational datasets (Advanced)
+>
+{: .checklist}
```

Comment:
I would say this list is already covered by the schedule that is rendered automatically at the bottom.

Reply:
I think it's good to be explicit and repeat this information.

Reply:
@ledm alright!